package/src/Slider.tsx

```diff
@@ -298,30 +291,36 @@ const SliderComponent = (
       onLayout={(event) => {
         setWidth(event.nativeEvent.layout.width);
       }}
-      style={[style, {justifyContent: 'center'}]}>
+      style={[style, {justifyContent: 'center'}]}
+    >
       {props.StepMarker || !!props.renderStepNumber ? (
         <StepsIndicator
           options={options}
           sliderWidth={width}
           currentValue={currentValue}
-          renderStepNumber={localProps.renderStepNumber}
-          thumbImage={localProps.thumbImage}
-          StepMarker={localProps.StepMarker}
-          isLTR={localProps.inverted}
+          renderStepNumber={props.renderStepNumber}
+          thumbImage={props.thumbImage}
+          StepMarker={props.StepMarker}
+          isLTR={inverted}
         />
       ) : null}
       <RCTSliderNativeComponent
-        {...localProps}
-        value={value}
+        {...props}
+        minimumValue={minimumValue}
+        maximumValue={maximumValue}
+        step={step}
+        inverted={inverted}
+        tapToSeek={tapToSeek}
+        value={passedValue}
```

Comment:
@BartoszKlonowski basically, the props that were being set by `defaultProps` and then passed through `localProps` were these 6 props, which are now being set by the default function parameters and explicitly passed here. The other 2 from `defaultProps`, `lowerLimit` and `upperLimit`, are already passed below